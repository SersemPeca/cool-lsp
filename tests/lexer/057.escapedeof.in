00000000: 2257 6861 7420 6861 7070 656e 7320 6966  "What happens if
00000010: 2049 2065 7363 6170 6520 616e 2045 4f46   I escape an EOF
00000020: 3f20 5c                                  ? \
