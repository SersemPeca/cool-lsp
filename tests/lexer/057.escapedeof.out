#1 INT_CONST 00000000
#1 ':'
#1 INT_CONST 2257
#1 INT_CONST 6861
#1 INT_CONST 7420
#1 INT_CONST 6861
#1 INT_CONST 7070
#1 INT_CONST 656
#1 OBJECTID e
#1 INT_CONST 7320
#1 INT_CONST 6966
#1 ERROR: String contains unescaped new line
#2 INT_CONST 00000010
#2 ':'
#2 INT_CONST 2049
#2 INT_CONST 2065
#2 INT_CONST 7363
#2 INT_CONST 6170
#2 INT_CONST 6520
#2 INT_CONST 616
#2 OBJECTID e
#2 INT_CONST 2045
#2 INT_CONST 4
#2 OBJECTID f46
#2 TYPEID I
#2 OBJECTID escape
#2 OBJECTID an
#2 TYPEID EOF
#3 INT_CONST 00000020
#3 ':'
#3 INT_CONST 3
#3 OBJECTID f20
#3 INT_CONST 5
#3 OBJECTID c
#3 ERROR: Invalid symbol "?"
#3 ERROR: Invalid symbol "\\"
